sdk/js/src/User.ts

```diff
@@ -38,6 +45,7 @@ export class DVCPopulatedUser implements DevCycleUser {
         options: DevCycleOptions,
         staticData?: StaticData,
         anonymousUserId?: string,
+        headerUserAgent?: string,
```

Comment:
accept a user agent from outside so that the serverside can tell us what the user agent should be